source/features/unreleased-commits.tsx

```diff
@@ -6,12 +6,13 @@ import {TagIcon} from '@primer/octicons-react';
 import features from '../feature-manager.js';
 import observe from '../helpers/selector-observer.js';
 import api from '../github-helpers/api.js';
-import {addAfterBranchSelector, buildRepoURL, cacheByRepo, getLatestVersionTag} from '../github-helpers/index.js';
+import {buildRepoURL, cacheByRepo, getLatestVersionTag} from '../github-helpers/index.js';
 import isDefaultBranch from '../github-helpers/is-default-branch.js';
-import getDefaultBranch from '../github-helpers/get-default-branch.js';
 import pluralize from '../helpers/pluralize.js';
-import {branchSelectorParent} from '../github-helpers/selectors.js';
```

Comment:
`addAfterBranchSelector` and `branchSelectorParent` is now only used on a  one feature.
Need to consider dropping these from helpers later.